src/components/course/routes/CourseAbout.jsx

```diff
@@ -13,7 +13,7 @@ import {
   useIsAssignmentsOnlyLearner,
   usePassLearnerCsodParams,
 } from '../../app/data';
-import ExpiredSubscriptionModal from '../../expired-subscription-modal';
+import BlockingSubscriptionExpirationModal from '../../expired-subscription-modal';
```

Comment:
nit/consider: I wonder if `CustomSubscriptionExpirationModal` would be a bit more explicit than the `Blocking` prefix to better reflect this expiration modal is custom and non-standard. For instance, if the standard expiration modal was updated to be blocking, then both the custom and standard modals would both be "blocking" despite still having different intents.